assets/main.js

```diff
@@ -5,3 +5,30 @@ import Tooltip from './vendor/bootstrap/js/src/tooltip.js';
 for (const element of document.querySelectorAll('[data-bs-toggle="tooltip"]')) {
   new Tooltip(element);
 }
+
+document.addEventListener("DOMContentLoaded", function () {
+  const copyBtns = document.querySelectorAll(".copy-btn");
+  copyBtns.forEach((btn) => {
```

Comment:
```suggestion
  document.querySelectorAll(".copy-btn").forEach((btn) => {
```

Pretty minor thing, but I guess we can loop directly here?